ortools/linear_solver/xpress_interface.cc

```diff
@@ -277,13 +277,12 @@ class MPCallbackWrapper {
     for (const std::exception_ptr& ex : caught_exceptions_) {
       try {
         std::rethrow_exception(ex);
-      } catch (std::exception& ex) {
+      } catch (std::exception &ex) {
```

Comment:
Could you try to run the following command on your PR to fix formatting ?
`clang-format --style=file -i ortools/linear_solver/xpress_interface.cc`

ref: https://packages.ubuntu.com/jammy/clang-format

Reply:
sure, must I do it for the test class too (it would make the new test less readable)?